configs/config.sample.yml

```diff
@@ -1,12 +1,16 @@
 global:
   port: 8080
   cache:
-    redis: kevin-test.jshtkz.ng.0001.use1.cache.amazonaws.com:6379
+    redis: redis-test.jshtkz.ng.0001.use1.cache.amazonaws.com:6379
 
 # List of supported chains.
 # The HTTP endpoint for a given chain is <host>:<port>/<chainName>.
 chains:
-  - chainName: ethereum
+  - chainName: mainnet
+    cache:
+      # Sets the ttl of set values in the cache.
+      # A ttl of zero will disable the cache.
+      ttl: 6s
```

Comment:
Originally had an `enabled` field as well, but I figured a `ttl` of 0 basically means we don't want the cache so I collapsed it down to just `ttl`.

Reply:
This means that we no longer have a default TTL for chains, but I think this is for the better.  We should be consciously setting TTL values for each chain that needs caching.  Each chain requires some research to choose a proper value.

Reply:
Sgtm. It looks like we’re not setting the key if it’s 0 so that’s good.